lms/templates/main.html

```diff
@@ -82,8 +82,10 @@
   % endif
   <script type="text/javascript" src="${static.url(jsi18n_path)}"></script>
   <script type="text/javascript" src="${static.url(ie11_fix_path)}"></script>
+  <% favicon_url = branding_api.get_favicon_url() %>
+  <link rel="icon" type="image/x-icon" href="${favicon_url}"/>
```

Comment:
Will this change revert once the logo is part of static assets?

Reply:
we are getting the url from `get_favicon_url` method, that method is responsible for returning absolute URLs to the favicon either local or remote.